### PR TITLE
Issue#325 correcting home icon alignment

### PR DESCRIPTION
**Pull request by jeromtom**
## Description
It is a simple bug causing alignment issues fixed by custom code targeting the element.

Fixes #325 raised in keploy main repository.
https://github.com/keploy/keploy/issues/325

## Type of change

- [X] Bug fix (non-breaking change which fixes an issue).

## How Has This Been Tested?
It has been tested locally.

## Additional Context (Please include any Screenshots/gifs if relevant)
Before:
DESKTOP
image

MOBILE
image

After:
DESKTOP
image

MOBILE
image

...

## Checklist:

- [X] My code follows the style guidelines of this project.
- [X] I have performed a self-review of my own code.

### CODE REVIEW

**Comment by jeromtom** [comment]
This PR is ready to merge.

**Comment by nehagup** [comment]
Thanks for the PR @jeromtom 


@SanskritiHarmukh can you please review? Please check on different screen sizes.

File: src/css/custom.css [hunk 1]
```diff
@@ -524,3 +524,8 @@ div[class^="announcementBar_"] {
 .list-disc > a {
   display: inherit;
 }
+
+/*Correcting alignment of home icon innavbar breadcrumbs*/
+a[class="breadcrumbs__link"]{
+  margin-bottom: -10px;
+}
```
Comment (SanskritiHarmukh) [inline src/css/custom.css:531]: Hey @jeromtom , It's slightly lower than the required position. Can you fix that?
Below are the results that I got when I tested the changes.

Desktop site
Screenshot from 2023-03-05 03-12-04

Normal
Screenshot from 2023-03-05 03-12-55

Phone
Screenshot from 2023-03-05 03-13-12

Comment (jeromtom) [reply src/css/custom.css:531]: Will try a margin of -8px

**Comment by jeromtom** [comment]
Forgot to sign off again. Rebasing branch.

**Comment by jeromtom** [comment]
Rebased PR to #150

**Pull request closed without merging**